tools/coverity/misra.config

```diff
@@ -10,26 +10,44 @@
             category: "Advisory",
             reason: "Allow inclusion of function like macros."
         },
+        {
+            deviation: "Rule 2.5",
+            reason: "Allow unused macros. Library headers may define macros intended for the application's use, but not used by a specific file."
```

Comment:
Do you have an example in mind?
I was thinking that what would be the use of that macro if the library is not using it? Maybe I am missing something.